src/utilities.jl

```diff
@@ -115,3 +115,18 @@ function replicate(f::Function, n::Integer; use_threads=false)
     end
     results
 end
+
+"""
+    testdata(nm)
+
+Return a data frame of test data set named `nm`, which can be a `String` or `Symbol`
+"""
+testdata(nm::AbstractString) = Feather.read(joinpath(TestData, nm * ".feather"))
```

Comment:
Following up on the suggestion of @Nosferican, we could rename both `testdatasets` and `testdata` to `datasets` and let the compiler figure out which methods we need.